src/getShadowNodeFromRef.web.ts

```diff
@@ -0,0 +1,7 @@
+// Used by GestureDetector (unsupported on web at the moment) to check whether the
```

Comment:
I would add this comment to non-web version too.

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/a94c858205b10551ee7e2383f0bcf55d3d73f81f.